packages/Chem/src/package.ts

```diff
@@ -483,25 +486,47 @@ export async function chemSpaceTopMenu(table: DG.DataFrame, molecules: DG.Column
     return;
   }
 
-  const embedColsNames = getEmbeddingColsNames(table);
+  const allowedRowCount = methodName === 'UMAP' ? 100000 : 10000;
+  const fastRowCount = methodName === 'UMAP' ? 5000 : 2000;
```

Comment:
can we put `'UMAP'` to const? we use it twice or more

Reply:
Done